src/satbias/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# (C) Copyright 2021 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+ecbuild_add_executable( TARGET  satbias2ioda.x
+                        SOURCES SatBiasConverter.cpp GsiSatbiasReader.cpp GsiSatbiasReader.h
+                        LIBS    Eigen3::Eigen ${oops_LIBRARIES} ioda_engines )
+
+ecbuild_add_test( TARGET  test_satbias_coding_norms
```

Comment:
I think the test name should be: `test_${PROJECT_NAME}_satbias_coding_norms`

Reply:
thank you; fixed.